zeppelin-web/src/app/interpreter/interpreter.controller.js

```diff
@@ -325,6 +325,15 @@ angular.module('zeppelinWebApp').controller('InterpreterCtrl',
         return;
       }
 
+      if (!$scope.newInterpreterSetting.name.indexOf('.') >= 0) {
```

Comment:
hi, maybe this is a bug.
`if ($scope.newInterpreterSetting.name.indexOf('.') >= 0) {`